spec/site_prism/page_objects/page/organisations/course_preview.rb

```diff
@@ -24,6 +24,8 @@ class CoursePreview < PageObjects::Base
         element :salary_details, "#section-salary"
         element :scholarship_amount, "[data-qa=course__scholarship_amount]"
         element :bursary_amount, "[data-qa=course__bursary_amount]"
+        element :early_career_payment_details, "[data-qa=course__early_career_payment_details]"
```

Comment:
Can we add the others here?

Reply:
This is a PageObject used for testing, therefore the only elements that should appear hear are the ones that used in the associated spec. The feature/preview_spec.rb tests the happy path only with a course that has scholarship, bursary and early career payment incentives.

Reply:
so `"[data-qa=course__loan_details]"` from
spec/views/preview.html.erb_spec.rb